tests/functional/adapter/test_grants.py

```diff
@@ -30,3 +31,17 @@ def assert_expected_grants_match_actual(self, project, relation_name, expected_g
         diff_a = BaseContext.diff_of_two_dicts(actual_grants, expected_grants)
         diff_b = BaseContext.diff_of_two_dicts(expected_grants, actual_grants)
         assert diff_a == diff_b == {}
+
+
+@pytest.mark.hive
+# TODO: setup Galaxy and Starbust tests
+#   See https://github.com/starburstdata/dbt-trino/issues/147
+#   and also https://github.com/starburstdata/dbt-trino/issues/146
+@pytest.mark.skip_profile("starburst_galaxy")
+@pytest.mark.skip(reason="Hive doesn't raise errors on invalid roles")
```

Comment:
To be investigated. Seems like Hive doesn't raise an error on invalid grants.

Also these exceptions might be different for Galaxy, SEP or HIve.

Reply:
Is it now skipped for Trino as well, if yes then `@pytest.mark.skip_profile("starburst_galaxy")` is obsolete.